tests/integration-tests/tests/multiple_nics/test_multiple_nics.py

```diff
@@ -15,7 +15,7 @@
 from utils import get_compute_nodes_instance_ids
 
 
-@pytest.mark.usefixtures("os", "instance", "scheduler")
+@pytest.mark.usefixtures("os", "instance", "scheduler","serial_execution_by_instance")
```

Comment:
minor: you should add an extra space after the comma (see GitHub linters checks)

Reply:
Fixing the Code linter Checks